creator-node/src/utils.js

```diff
@@ -227,21 +227,24 @@ const ipfsGet = ({ ipfsLatest }, logger, path, timeout = 1000) => new Promise(as
   }
 })
 
-async function findCIDInNetwork (filePath, cid, logger, libs, trackId = null) {
+async function findCIDInNetwork (filePath, cid, logger, libs, trackId = null, excludeList = []) {
   const attemptedStateFix = await getIfAttemptedStateFix(filePath)
   if (attemptedStateFix) return
 
   // get list of creator nodes
   const creatorNodes = await getAllRegisteredCNodes(libs)
   if (!creatorNodes.length) return
 
+  // Remove excluded nodes from list of creator nodes, no-op if empty list or nothing passed in
+  const creatorNodesFiltered = creatorNodes.filter(c => excludeList.includes(c.endpoint))
```

Comment:
shouldn't this be `!excludeList.includes`

Reply:
ah yes you are correct! sorry its been a long day lol

Reply:
hahaha all good dude - would be good to make sure new code path still works

also i re-ran maddog ci since it failed for non-relevant reason